### PR TITLE
chore: remove duplicate containerd task collecion

### PR DESCRIPTION
**Pull request by mainred**

### CODE REVIEW

File: vhdbuilder/scripts/windows/collect-windows-logs.ps1 [hunk 1]
```diff
@@ -116,6 +103,19 @@ else {
   Write-Host "ctr.exe command not avaiable"
 }
 
+# Containerd log is outside the c:\k folder
```
Comment (mainred) [inline vhdbuilder/scripts/windows/collect-windows-logs.ps1:106]: This change is to group containerd logs

Comment (AbelHu) [inline vhdbuilder/scripts/windows/collect-windows-logs.ps1:106]: Containerd **panic** log

File: vhdbuilder/scripts/windows/collect-windows-logs.ps1 [hunk 2]
```diff
@@ -132,15 +132,6 @@ else {
   Write-Host "Calico logs not avalaible"
 }
 
-# log containerd containers (this is done for docker via networking collectlogs.ps1)
-$res = Get-Command ctr.exe -ErrorAction SilentlyContinue
```
Comment (AbelHu) [inline vhdbuilder/scripts/windows/collect-windows-logs.ps1:136]: Will these be also collected by collectlogs.ps1 so we can remove it?

Comment (AbelHu) [reply vhdbuilder/scripts/windows/collect-windows-logs.ps1:136]: Found it. I did not find the duplicate in the file but the file exists in your branch

**Review by AbelHu — APPROVED** [review]

File: vhdbuilder/scripts/windows/collect-windows-logs.ps1 (hunk 2 above)
Comment (mainred) [reply vhdbuilder/scripts/windows/collect-windows-logs.ps1:136]: It is collected in L90-L104

**Pull request merged**